modin/pandas/dataframe.py

```diff
@@ -2229,6 +2229,21 @@ def to_parquet(
             **kwargs,
         )
 
+    def to_ray_dataset(self):
```

Comment:
What about Series?

Reply:
I think it is okay to have a one column DataFrame out of a one column Ray Dataset. If the user needs a series, just call .squeeze(axis=1).

Reply:
> Many pandas operations return different types depending on the data, so we force users to check the type of the result of any operation every time.

I missed understanding. What Ray type do you suggest converting Series instead of Dataset?

Reply:
`modin.pandas.Series` doesn't have `to_ray_dataset` but `modin.pandas.DataFrame` has. In order not to receive errors such as the absence of an attribute on an object, the user will have to check the type.

Reply:
Ok, thanks for your clarification. I think it will be really more convenient if we add `to_ray_dataset` function to `modin.pandas.Series`

Reply:
Done